### PR TITLE
Allow Debug messages from ec2 driver to include request / response

### LINKED ISSUE #2794: stdout from driver
the ec2 driver now uses the amazon ec2 go library.  As a side affect to this I am no longer able to monitor the aws calls in debug mode.  I am assuming that this is due to the new driver model not displaying stdout. Is my assumption correct?  Is there any way around this.  I see the ec2 library allows the implementation of a custom logger,  is using the libmachine/log with a customer logger my only option?

### PR DESCRIPTION
**Pull request by jeffellin**
Allow Debug messages from ec2 driver to include request / response data sent to aws.

This Closes #2794 

Signed-off-by: Jeffrey Ellin <jeff@ellin.com>

### CODE REVIEW

File: drivers/amazonec2/amazonec2.go [hunk 1]
```diff
@@ -655,8 +655,11 @@ func (d *Driver) Remove() error {
 
 func (d *Driver) getClient() *ec2.EC2 {
 	config := aws.NewConfig()
+	alogger := AwsLogger()
 	config = config.WithRegion(d.Region)
 	config = config.WithCredentials(credentials.NewStaticCredentials(d.AccessKey, d.SecretKey, d.SessionToken))
+	config = config..WithLogger(alogger)
```
Comment (nathanleclaire) [inline drivers/amazonec2/amazonec2.go]: Looks like you have an extra `.` on these lines

Comment (jeffellin) [inline drivers/amazonec2/amazonec2.go]: odd it built locally,  fix coming.

**Comment by nathanleclaire** [comment]
LGTM

@docker/machine-maintainers PTAL

**Comment by dgageot** [comment]
LGTM

**Pull request merged**